### PR TITLE
Updates to service advertisment usage doc

### PR DESCRIPTION
**Pull request by caseydavenport**
## Description

Goals of this PR are to:
- Make it a bit more clear when / why this should be enabled.
- Make it more clear what value to use when setting the `CALICO_ADVERTISE_CLUSTER_IPS` var.
- Switch up the "behavior" section so that behavior comes first, implementation comes second. 

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 53b7ac30fa05a8944889a5de0f62e977e02cde22

https://deploy-preview-2315--calico.netlify.com

**Comment by 2ffs2nns** [comment]
much better @caseydavenport

File: master/usage/service-advertisement.md [hunk 1]
```diff
@@ -3,33 +3,61 @@ title: Advertising Kubernetes services over BGP
...
+## Prerequisites
 
--  for each active service with `externalTrafficPolicy: Local`, the
-   cluster IP for that service is advertised as a /32 route from the
-   nodes that have endpoints for that service.
+**BGP peers**
+
+You must have established BGP connections to one or more routers outside of your Kubernetes cluster.
+See [configuring BGP peers][bgp-peers] for more information.
+
```
Comment (neiljerram) [inline master/usage/service-advertisement.md]: Should we add something here about those external routers needing to have the equivalent of BIRD's `merge paths on;` config?  (i.e. configuration to program ECMP routes into their local kernel)

File: master/usage/service-advertisement.md [hunk 2]
```diff
@@ -3,33 +3,61 @@ title: Advertising Kubernetes services over BGP
...
+Then, set the [`CALICO_ADVERTISE_CLUSTER_IPS` environment variable][reference] in the {{site.noderunning}} daemon set
+equal to the service cluster IP range for the cluster.
+
+The following command will enable service cluster IP advertisement on a running cluster, assuming the default cluster IP range
+of `10.0.0.0/24`.
+
+```bash
+kubectl patch ds -n kube-system calico-node --patch \
+    '{"spec": {"template": {"spec": {"containers": [{"name": "calico-node", "env": [{"name": "CALICO_ADVERTISE_CLUSTER_IPS", "value": "10.0.0.0/24"}]}]}}}}’
```
Comment (neiljerram) [inline master/usage/service-advertisement.md]: Can this be done with YAML and `<< EOF`?  If so, would that look clearer?

File: master/usage/service-advertisement.md (hunk 1 above)
Comment (caseydavenport) [reply master/usage/service-advertisement.md]: Yep, definitely.

**Pull request merged**